src/server/ApiRoutes/Hold.js

```diff
@@ -90,6 +118,8 @@ function getDeliveryLocations(barcode, patronId, accessToken, cb, errorCb) {
     }
   )
   .then(barcodeAPIresponse => {
+    mapLocationDetails(barcodeAPIresponse.data.itemListElement[0].deliveryLocation);
```

Comment:
This function returns an array so you should assign the result to a variable and then use it. Right now it's implicitly making changes to the `barcodeAPIresponse` variable.